templates/shopapi/base.html

```diff
@@ -3,26 +3,37 @@
 <!DOCTYPE html>
 <html>
 <head>
-<meta name="viewport" content="width=device-width, initial-scale=1">
-<link rel="stylesheet" type="text/css" href="{% static './css/base_page.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static './css/base_page.css' %}">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" rel="stylesheet"/>
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet"/>
 </head>
-<body>
+    <body>
+        <div class="header">
+            <a class="active" href="{% url 'home' %}" class="logo">I-SHOP</a>
+            <div class="header-right">
+            <a href="{% url 'products' %}">Products</a>
+            {% if user.is_authenticated %}
+              <a href="{% url 'logout' %}">Logout</a>
+            {% else %}
+              <a href="{% url 'signup' %}">Sign Up</a>
+              <a href="{% url 'login' %}">Log In </a>
+            {% endif %}
+            </div>
+        </div>
+        <div class="content">
+            {% block content %}
+            {% endblock %}
+        </div>
+        <div class="footer">
```

Comment:
@vasyltsar we need responsive template. Your footer is not in the bottom of the screen.
Here is example how to implement it. Very simple solution
https://stackoverflow.com/questions/7527152/div-with-dynamic-min-height-based-on-browser-window-height